src/Kros.AspNetCore/Extensions/ConfigurationExtensions.cs

```diff
@@ -26,5 +28,21 @@ public static class IConfigurationExtensions
         /// <returns>Section.</returns>
         public static T GetSection<T>(this IConfiguration configuration, string sectioName) where T : class
             => configuration.GetSection(sectioName).Get<T>();
+
+        /// <summary>
+        /// Gets allowed origins setting from appSettings.json.
+        /// </summary>
+        /// <param name="configuration">The configuration.</param>
+        public static string[] GetAllowedOrigins(this IConfiguration configuration)
+        {
+            Type corsSectionType = configuration.GetSection(CorsOptions.CorsSectionName).GetType();
```

Comment:
`configuration.GetSection(CorsOptions.CorsSectionName)` is called three times. Maybe more readable will be to store and use this in variable and call '.GetType()' only in `if`.